src/stories/5-custom/array-field/array-field.stories.tsx

```diff
@@ -154,37 +157,55 @@ const SCHEMA_NESTED_ARRAY: Record<string, TFrontendEngineFieldSchema> = {
 				columns: { mobile: 4, tablet: 8, desktop: 12 },
 				validation: [{ required: true }],
 			},
-			color: {
-				uiType: "select",
-				label: "Color",
-				options: [{ label: "Red", value: "Red" }],
+			email: {
+				uiType: "email-field",
+				label: "Email",
 				columns: { mobile: 4, tablet: 4, desktop: 6 },
 			},
-			testArray: {
-				referenceKey: "array-field",
-				sectionTitle: "Nested array",
+			uinfin: {
+				uiType: "masked-field",
+				label: "Uinfin",
+				columns: { mobile: 4, tablet: 4, desktop: 6 },
+			},
+			wrapper: {
+				uiType: "div",
+				style: {
+					padding: "1rem",
+					borderRadius: 4,
+					border: `1px solid #E0E4E5`,
+				},
 				columns: { mobile: 4, tablet: 8, desktop: 12 },
-				fieldSchema: {
-					grid: {
-						uiType: "grid",
-						style: { marginTop: 16, marginBottom: 16 },
-						children: {
-							name: {
-								uiType: "text-field",
-								label: "Name",
-								columns: { mobile: 4, tablet: 8, desktop: 12 },
-								validation: [{ required: true }],
-							},
-							color: {
-								uiType: "select",
-								label: "Color",
-								options: [{ label: "Red", value: "Red" }],
-								columns: { mobile: 4, tablet: 4, desktop: 6 },
+				children: {
+					testArray: {
+						referenceKey: "array-field",
+						sectionTitle: "Nested array",
+						columns: { mobile: 4, tablet: 8, desktop: 12 },
+						fieldSchema: {
+							grid: {
+								uiType: "grid",
+								style: { marginTop: 16, marginBottom: 16 },
+								children: {
+									name: {
+										uiType: "text-field",
+										label: "Nested name",
+										columns: { mobile: 4, tablet: 8, desktop: 12 },
+										validation: [{ required: true }],
+									},
+									email: {
+										uiType: "email-field",
+										label: "Nested email",
+										columns: { mobile: 4, tablet: 4, desktop: 6 },
+									},
+									uinfin: {
+										uiType: "masked-field",
+										label: "Nested uinfin",
+										columns: { mobile: 4, tablet: 4, desktop: 6 },
+									},
+								},
```

Comment:
remember to update error object in the story
```suggestion
							grid: {
								uiType: "grid",
								children: {
									childName: {
										uiType: "text-field",
										label: "Child name",
										columns: { mobile: 4, tablet: 8, desktop: 12 },
										validation: [{ required: true }],
									},
									childEmail: {
										uiType: "email-field",
										label: "Child email",
										columns: { mobile: 4, tablet: 4, desktop: 6 },
									},
									childUinfin: {
										uiType: "masked-field",
										label: "Child uinfin",
										columns: { mobile: 4, tablet: 4, desktop: 6 },
									},
								},
							},
```